scripts/dts/extract/globals.py

```diff
@@ -144,7 +145,8 @@ def get_reduced(nodes, path):
 def get_node_label(node_compat, node_address):
     def_label = convert_string_to_label(node_compat.upper())
     if '@' in node_address:
-        def_label += '_' + node_address.split('@')[-1].upper()
+        def_label += '_' + \
+                convert_string_to_label(node_address.split('@')[-1].upper())
```

Comment:
Following recent merge of ba2ce2e9f0dcccef8b14c0aea2bf5e483ee07f82, remove the call to ``.upper()``